women-in-tech-resources/resources-list.md

```diff
@@ -76,30 +81,34 @@ If this is your first time looking at the Women in Tech Resources List, spend 30
 - [Girls In Tech India – Catalyst Conference](https://india.girlsintech.org/)
 - [IEEE WiE](https://wie.ieee.org/)
 - [Facebook Developer Circle - Various States](https://developers.facebook.com/developercircles)
-
-### Meetups and Events for Women & Non-binary Engineers:
-- [Women Who Code - Delhi](https://www.womenwhocode.com/delhi)
-- [Google Developers Group - Various States](https://developers.google.com/programs/community/)
-- [Paytm - Build For India](https://www.meetup.com/Paytm-Build-for-India/)
-- [PyDelhi - Delhi](https://www.meetup.com/pydelhi/events/254577423/)
-- [Mozilla Campus Clubs - Various States](https://campus.mozilla.community/)
-- [Women Techmakers](https://www.womentechmakers.com/india)
-
-### Conferences & Hackathons
-- [Hackference](https://hackference.in/)
 - [Surge](https://surgeconf.com/surge2019)
 - [Tie Global Summit](http://www.tieglobalsummit.org/)
 
+### Hackathons
+- [Hackference](https://hackference.in/)
+- [Linkedin Wintathon](https://wintathon2019.splashthat.com/)
+- [HackInOut](https://hackinout.co/)
+- [Smart India Hackathon](https://www.sih.gov.in/)
+
 ### Open Source Programs
 - [Google Summer of Code - Global](https://summerofcode.withgoogle.com/)
 - [GirlScript Summer Of Code](https://gssoc.tech/)
 - [Outreachy](https://www.outreachy.org/)
 - [Rails Girls Summer of Code](https://railsgirlssummerofcode.org/)
+- [Hacktoberfest](https://hacktoberfest.digitalocean.com/)
```

Comment:
Could you put hacktoberfest under the Bay Area topic too? We were part of that last year!